backend/src/main/resources/db/migration/V1_0_11__Change_Column_Rejected_To_Confirmed.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE volunteers_proposals RENAME COLUMN rejected TO confirmed;
+
+
```

Comment:
You should not create 3 files in the same PR whihc are undoing the changes of the previous files. Create just the final version

Reply:
Done